inference-engine/src/inference_engine/src/ie_itt.hpp

```diff
@@ -17,6 +17,14 @@ namespace domains {
     OV_ITT_DOMAIN(IE);
     OV_ITT_DOMAIN(IE_LT);
     OV_ITT_DOMAIN(IE_RT);
-}
-}
-}
+}  // namespace domains
+}  // namespace itt
+}  // namespace InferenceEngine
+
+namespace ov {
+namespace itt {
+namespace domains {
+    OV_ITT_DOMAIN(IE);
+}  // namespace domains
```

Comment:
can we use `ov::itt::domains` even in old places?

Reply:
Fixed